static/src/js/util/variables.js

```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line arrow-body-style
```

Comment:
This isn't necessary, you can still do multiline (without the `{}`) and prevent having to use this eslint-disable here, I think.